physionet-django/console/forms.py

```diff
@@ -51,10 +51,10 @@
 
 YES_NO_NA_UNDETERMINED = (
     ('', '-----------'),
-    (1, 'Yes'),
     (0, 'No'),
+    (1, 'Yes'),
```

Comment:
Changing the order here (or perhaps the change in the model) has affected the default order of responses in the form, so "no" appears before "yes". It would be better if these fields were consistent, with "yes" appearing first.

Reply:
Thanks for taking a look, I switched the "No" and the "Yes"!